bin/worker.js

```diff
@@ -31,11 +33,14 @@ export default {
       return new Response('Missing required fields', { status: 400 })
     }
 
+    // Timestamp to measure file retrieval performance
+    const t1 = performance.now()
```

Comment:
I think we should measure the time for error responses too, shouldn't we?

(See `return new Response('Missing required fields', { status: 400 })` on the line above - when we return from the handler early, timing is not recorded.)

Reply:
I'd like to see the number of all requests on our dashboard, not only the number of "valid" requests.

I created a follow-up issue for this: https://github.com/filcdn/roadmap/issues/272